bin/main.rb

```diff
@@ -42,32 +50,39 @@ def board
   $arr = [[' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' '], [' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' '], [' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ']]
 end
 
+# Game
 class Game
```

Comment:
- [ ] All of the game-logic resources (class/modules) should be in the lib directory and not in the bin directory.
screenshot of the requirements 👇 
![Screen Shot 2020-10-01 at 11 05 47 PM](https://user-images.githubusercontent.com/48804258/94863374-a652f180-043a-11eb-9db2-24d2bb5d0392.png)